components/02-molecules/menus/social-menu/_social-menu.scss

```diff
@@ -0,0 +1,31 @@
+.social-menu {
+  @include list-reset;
+}
+
+.social-menu__item {
+  margin-bottom: $space-one-half;
+}
+
+.social-menu__link {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  text-decoration: none;
+  color: clr(text);
+
+  &:focus,
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+.social-menu__icon {
+  height: 2rem;
```

Comment:
Per our group discussion: Lets create an `$social-icon-size` size variable at the top of this file then set the height/width to that variable.